vmms/distDocker.py

```diff
@@ -0,0 +1,313 @@
+#
+# distDocker.py
+#
+# Implements the Tango VMMS interface to run Tango jobs in 
+# docker containers on a list of host machines. This list of
+# host machines must be able to run docker and be accessible
+# by SSH. The IP address of the host machine is stored in the
+# `domain_name` attribtue of TangoMachine.
+#
+
+import random, subprocess, re, time, logging, threading, os, sys, shutil
+import config
+from tangoObjects import TangoMachine
+
+def timeout(command, time_out=1):
```

Comment:
So I didn't catch this for the last PR, but is there a reason why the timeout function is not in a library somewhere? It looks like `timeout` and `timeoutWithReturnStatus` are just copied between all the vmms's.